packages/edit-site/src/components/block-editor/index.js

```diff
@@ -108,8 +109,8 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 					}
 				} }
 			>
+				<BlockEditorKeyboardShortcuts.Register />
```

Comment:
Should we register this alongside other shortcuts, to match other editors? https://github.com/WordPress/gutenberg/blob/b6400f7d3ed330976ee074b252554c2ae7cea738/packages/edit-site/src/components/editor/index.js#L185?

Reply:
IMO is better to have them here, in the same place where we add the `block-editor` in site editor, but in general 🤷 😄

Reply:
No strong opinion on this 😄  Feel free to merge.